### ISSUE TITLE
unexpected Content-Type: "application/vnd+hashicorp.releases-api.v0+json"

### ISSUE THREAD

**Issue by tekumara** [hashicorp/terraform-plugin-docs#170]
```
$ go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
rendering website for provider "terraform-provider-prefect"
exporting schema from Terraform
compiling provider "prefect"
getting Terraform binary
Error executing command: unable to generate website: unexpected Content-Type: "application/vnd+hashicorp.releases-api.v0+json"

exit status 1
```

go.mod contains:
```
	github.com/hashicorp/terraform-plugin-docs v0.6.0
	github.com/hashicorp/terraform-plugin-framework v0.6.0
	github.com/hashicorp/terraform-plugin-go v0.8.0
	github.com/hashicorp/terraform-plugin-sdk/v2 v2.10.1
```

**Comment by tekumara** [comment]
Related https://github.com/hashicorp/hc-install/pull/57

**Comment by jacobbednarz** [comment]
this was fixed long ago by #135; you need to update to the newer versions to address it.

**Comment by tekumara** [comment]
Thanks @jacobbednarz I've resolved this by upgrading to:
```
	github.com/hashicorp/terraform-plugin-docs v0.13.0
	github.com/hashicorp/terraform-plugin-framework v0.10.0
	github.com/hashicorp/terraform-plugin-go v0.12.0
	github.com/hashicorp/terraform-plugin-sdk/v2 v2.19.0
```
